example/spec/serial/DiscoverGranulesHttpSpec.js

```diff
@@ -113,23 +113,14 @@ describe('The Discover Granules workflow with http Protocol', () => {
 
     await Promise.all(discoverGranulesLambdaOutput.payload.granules.map(
       async (granule) => {
-        try {
-          await waitForApiStatus(
-            getGranule,
-            {
-              prefix: config.stackName,
-              granuleId: granule.granuleId,
-            },
-            'completed'
-          );
-        } catch (error) {
-          console.log(`Error waiting for API status on granule ${JSON.stringify(granule)}`);
-          throw error;
-        }
-        await deleteGranule({
-          prefix: config.stackName,
-          granuleId: granule.granuleId,
-        });
+        await waitForGranuleAndDelete(
+          config.stackName,
+          granule.granuleId,
+          'completed',
+          {
+            retries: 5,
```

Comment:
Just curious if we know why the retry behavior needed to be updated here

Reply:
Nope, removed it.